src/themes/designer/styles/workshops.scss

```diff
@@ -40,6 +40,8 @@
         h1 {
             padding-left: 5px;
             line-height: 25px;
+
+         
```

Comment:
I think we can leave these lines deleted

Reply:
@malincrist, I'll be merging update today, I'll remove those lines as well.